src/NodaTime/LocalDate.cs

```diff
@@ -198,10 +198,18 @@ public LocalDate(Era era, int yearOfEra, int month, int day, CalendarSystem cale
         /// by this value.
         /// </summary>
         /// <remarks>
+        /// <para>
         /// <see cref="DateTimeKind.Unspecified"/> is slightly odd - it can be treated as UTC if you use <see cref="DateTime.ToLocalTime"/>
         /// or as system local time if you use <see cref="DateTime.ToUniversalTime"/>, but it's the only kind which allows
         /// you to construct a <see cref="DateTimeOffset"/> with an arbitrary offset, which makes it as close to
         /// the Noda Time non-system-specific "local" concept as exists in .NET.
+        /// </para>
+        /// <para>
+        /// <see cref="DateTime"/> does not contain any state for the calendar system of the value.
```

Comment:
I wonder if another way to put this would be that `DateTime` uses the Gregorian calendar by definition, and so the value will be effectively converted to the Gregorian calendar first, before being represented as a `DateTime`?

Reply:
Yup. I'll make a first stab at the change and merge on green; we can always iterate on the precise wording.